react/features/toolbox/components/web/Toolbox.tsx

```diff
@@ -1394,7 +1405,7 @@ class Toolbox extends Component<Props> {
  * @private
  * @returns {{}}
  */
-function _mapStateToProps(state, ownProps) {
+function _mapStateToProps(state: any, ownProps: Partial<Props>) {
```

Comment:
no IState?

Reply:
Not all the features used are typed yet (`features/virtual-background`).